src/ontology/uberon.Makefile

```diff
@@ -1259,8 +1259,9 @@ ifeq ($(strip $(IMP)),true)
 
 # FBbt mapping set. Nominally a simple mirror, but we need a custom rule
 # because the default, ODK-generated rule would ignore the MIR variable.
-$(MAPPINGDIR)/fbbt.sssom.tsv:
-	wget "http://purl.obolibrary.org/obo/fbbt/fbbt-mappings.sssom.tsv" -O $@
+.PHONY: fbbt-mappings
```

Comment:
Usually I recomment to use .FORCE which is a widely used hack for this and a bit nicer for documentation purposes (no need to check what fbbt-mappings does) 

https://seanbone.ch/makefile-hacks/#Forcing_a_target_to_rebuild

Reply:
Adopted, thanks!